src/app/wallets/get-transactions-for-wallet.ts

```diff
@@ -42,8 +50,12 @@ export const getTransactionsForWallets = async (
 
   // we are getting both the transactions in the mempool and the transaction that
   // have been mined by not yet credited because they haven't reached enough confirmations
-  const onChainTxs = await onChain.listIncomingTransactions(ONCHAIN_MIN_CONFIRMATIONS)
-  if (onChainTxs instanceof OnChainError) {
+  const onChainTxs = await redisCache.getOrSet({
+    key: CacheKeys.LastOnChainTransactions,
+    ttlSecs: toSeconds(SECS_PER_5_MINS * 2),
```

Comment:
`SECS_PER_5_MINS * 2` is not great imo. if we want 10 min we could have a const for it?

Reply:
fixed here https://github.com/GaloyMoney/galoy/pull/1383/commits/523bef1133049ece9c516024caa82d744448a241